build/Publish/PublishNupkgToTransportFeed.targets

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'Common.props'))\Common.props" />
+
+  <PropertyGroup>
+    <!-- Capture 'OutputPath' before Publish.csproj modifies it -->
+    <OutputPathTransportFeed>$(OutputPath)</OutputPathTransportFeed>
+    <BuildTasksFeedDllVersion>1.0.0-prerelease-01929-02</BuildTasksFeedDllVersion>
```

Comment:
Personally, I'd move this into the csproj (before the import) so that you can see the package reference and version in the same place rather than having to go the targets file to see what version is used.  I view the targets file more as supporting plumbing for the toggles defined in the csproj.

Reply:
Good idea...done.